openassessment/templates/openassessmentblock/edit/oa_edit_prompt.html

```diff
@@ -0,0 +1,27 @@
+{% load i18n %}
+{% spaceless %}
+<li class="openassessment_prompt is-collapsible" data-prompt="{{ prompt_uuid }}">
+
+    <div class="openassessment_container_header openassessment_prompt_header view-outline">
+        <a class="action expand-collapse collapse"><i class="icon-caret-down ui-toggle-expansion"></i></a>
+        <div class="openassessment_container_header_title_box openassessment_prompt_header_title_box">
+            <h6 class="openassessment_container_header_title openassessment_prompt_header_title">{% trans "Prompt" %}</h6>
+            <p class="openassessment_container_guide openassessment_prompt_guide">{% trans "You cannot delete a prompt after the assignment has been released." %}</p>
+        </div>
+        <div class="openassessment_container_remove_button openassessment_prompt_remove_button"><h2>{% trans "Remove" %}</h2></div>
```

Comment:
I know you are just imitating the structure of the criterion remove here, but this remove button should be a button or a link, not an h2. This can be logged as a follow-up story if it is beyond the scope of this work.